content/docs/tasks/traffic-management/egress/index.md

```diff
@@ -264,6 +264,102 @@ $ export SOURCE_POD=$(kubectl get pod -l app=sleep -o jsonpath={.items..metadata
+and you did not define a `ServiceEntry` on port 443, you can access any external service on port 443. Note, however,
+that once you create an HTTP service on port 443 or define any `ServiceEntry` for any host on
+port 443, all accesses to port 443 is blocked: Istio will fall back to the blocking-by-default behavior for that
+port only. (Defining an HTTP service on port 443 is not recommended anyway, since using the same port for TCP/HTTPS and
+for HTTP traffic in Istio is discouraged.)
+
+1.  Make a couple of requests to external HTTPS services from `SOURCE_POD` to verify that they are blocked (it means you
+    did not configure `ServiceEntry` and `VirtualService` for the external services and that you did not enable
+    direct access to external services).
+
+    {{< text bash >}}
+    $ kubectl exec -it $SOURCE_POD -c sleep -- curl -s https://www.google.com | grep -o "<title>.*</title>"; kubectl exec -it $SOURCE_POD -c sleep -- curl -s https://edition.cnn.com | grep -o "<title>.*</title>"
+    command terminated with exit code 35
+    command terminated with exit code 35
+    {{< /text >}}
+
+1.  To allow access to all the external services, install or update Istio by using
+[Helm](https://preliminary.istio.io/docs/setup/kubernetes/helm-install/) while setting the value of
+`global.outboundTrafficPolicy.mode` to `ALLOW_ANY`: `--set global.outboundTrafficPolicy.mode=ALLOW_ANY`.
+
+    Alternatively, if you followed the instructions in
+    [Quick Start with Kubernetes](https://preliminary.istio.io/docs/setup/kubernetes/quick-start/#installation-steps)
+    and used `install/kubernetes/istio-demo.yaml` or `install/kubernetes/istio-demo-auth.yaml` files to install Istio,
+    just edit the files. Look for the following YAML part:
+
+    {{< text yaml >}}
+    # Set the default behavior of the sidecar for handling outbound traffic from the application:
+    # REGISTRY_ONLY - restrict outbound traffic to services defined in the service registry as well
+    #   as those defined through ServiceEntries
+    # ALLOW_ANY - outbound traffic to unknown destinations will be allowed, in case there are no
+    #   services or ServiceEntries for the destination port
+    outboundTrafficPolicy:
+      mode: REGISTRY_ONLY
+    {{< /text >}}
+
+    Change `mode` of `outboundTrafficPolicy` from `REGISTRY_ONLY` to `ALLOW_ANY`. Then run `kubectl apply` with the
+    edited file.
+
+    Yet another option is to use `kubectl edit` to edit the relevant configuration map directly:
+    `kubectl edit configmap istio -n istio-system`.
+
+    For demonstration purposes, let's use `kubectl get --export`, `kubectl replace` and `sed`:
+
+    {{< text bash >}}
+    $ kubectl get configmap istio -n istio-system --export -o yaml | sed 's/mode: REGISTRY_ONLY/mode: ALLOW_ANY/g' | kubectl replace -n istio-system -f -
+    configmap "istio" replaced
+    {{< /text >}}
```

Comment:
Does user need to restart pilot or anything?

Reply:
No, pilot now listens to the config map changes, so the user does not have to do anything, only has to update the config map.

Reply:
awesome!